dotcom-rendering/src/model/enhance-dividers.ts

```diff
@@ -1,6 +1,7 @@
 import { JSDOM } from 'jsdom';
 
-const isDinkus = (element: CAPIElement): boolean => {
+const isDinkus = (element?: CAPIElement): boolean => {
+	if (!element) return false;
```

Comment:
It felt cleaner to rework the signature of this method rather than extracting the previous element below.

Reply:
I opted for defining a previous element, in the end.